packages/react/src/__test__/ErrorBoundary.spec.tsx

```diff
@@ -140,4 +140,18 @@ describe('ErrorBoundary', () => {
     expect(container.textContent).not.toBe(ERROR_MESSAGE)
     expect(onReset).toHaveBeenCalledTimes(1)
   })
+
+  it('should throw error with no fallback', () => {
+    const Throw = () => {
+      throw new Error('something error')
+    }
+
+    expect(() =>
+      render(
+        <ErrorBoundary fallback={undefined}>
```

Comment:
I tried after your comment, I think other way is needed for it.

Omit cannot accept undefined or null because of generic's contrainting by `string | number | symbol`

```ts
type Omit<T, K extends string | number | symbol> = { [P in Exclude<keyof T, K>]: T[P]; }
```

Reply:
I made [this Pull Request](https://github.com/suspensive/react/pull/122) for this

Reply:
> Omit cannot accept undefined or null because of generic's contrainting by string | number | symbol

I didnt know that! Thanks for giving learning

Reply:
I didn't know that too! I learned every time with your review